platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/events/EventMetadata.java

```diff
@@ -64,6 +66,10 @@ public class EventMetadata extends AbstractHashable {
      * list of transactions
      */
     private final List<TransactionWrapper> transactions;
+    /**
+     * The bytes of the transactions
+     */
+    private final List<Bytes> transactionsBytes;
```

Comment:
I don't see this being needed, since the bytes will be inside the `TransactionWrapper`

Reply:
You're right. I removed it.